server/services/lifecycle/lifecycle.js

```diff
@@ -106,10 +106,8 @@ module.exports = ({ strapi }) => {
               )
             })
         },
-        async afterDeleteMany() {
-          strapi.log.error(
-            `Meilisearch could not find an example on how to access the \`afterDeleteMany\` hook. Please consider making an issue to explain your use case`
-          )
+        async afterDeleteMany(event) {
+          this.afterDelete(event)
```

Comment:
Is there any way to test this feature?

Reply:
Unfortunately it would be very hard to test because of the structure of the subscription.
In the previous version of Strapi v3 I used to try it in the cypress tests, but these are on their way:

https://github.com/meilisearch/strapi-plugin-meilisearch/issues/351
https://github.com/meilisearch/strapi-plugin-meilisearch/issues/103